src/config.rs

```diff
@@ -27,6 +27,22 @@ use io::Bundle;
 use status::StatusBackend;
 
 
+/// Awesome hack time!!!
+///
+/// This is part of the "test mode" described in the `test_util` module. When
+/// test mode is activated in this module, the `default_bundle()` and
+/// `format_cache_path()` functions return results pointing to the test asset
+/// tree, rather than whatever the user has actually configured.
+static mut CONFIG_TEST_MODE_ACTIVATED: bool = false;
```

Comment:
An `AtomicBool` here would avoid the few bits of unsafe code. Not sure if it's worth it though. `AtomicBool`s are safe to mutate by reference but slightly awkward to use.

Reply:
I'd also suggest using `AtomicBool` to avoid any unnecessary `unsafe`. Using it is not that bad. 
Define using `static CONFIG_TEST_MODE_ACTIVATED: AtomicBool = AtomicBool::new(false);`.
Reading is as easy as `CONFIG_TEST_MODE_ACTIVATED.load(Ordering::SeqCst)` and storing `CONFIG_TEST_MODE_ACTIVATED.store(forced, Ordering::SeqCst)`.

Reply:
Yes, I agree that this is better. Thanks for the suggestion!